### ISSUE TITLE
[A11Y] [Web] Unable to select date input clearing button

### ISSUE THREAD

**Issue by shengoogle** [ampproject/ampstart#415]
ENVIRONMENT:
Build number: 56.0.2924.87 beta (64-bit)
Form factor: Chromebook
Browser: Chrome

PRE-REQUISITE
chromevox is ON

STEPS TO REPRODUCE:
1. Open https://ampstart.com
2. Navigate to Components Tab
3. Using chromevox, navigate to Date input

ACTUAL RESULTS:
Using chromevox, navigate to the Date Input on the Components tab.  User is unable to navigate to clear date of expiry button.

EXPECTED RESULTS:
User should be able to navigate to clear date button and select it.

ATTACHMENT:
Please see the video in the link below for reference.

https://drive.google.com/open?id=0BwUNSMCBrCB4QXdPbVd5RVJ0SWs

**Comment by torch2424** [comment]
Hello,

Unfortunately, I do not think we can solve this issue with AMP Start as of now. We are currently using the [HTML 5 Standard <input type="date">](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date), in the current AMP Start date field. And there are no other standard alternatives outside of this, that are not a library that AMP would support. However, [HTML 5 date inputs aren't very accessible](https://ux.stackexchange.com/questions/60884/best-way-for-date-field-for-visually-impaired-users). And after testing all the possible ways of building HTML 5 datepickers at [this handy website](https://thepaciellogroup.github.io/AT-browser-tests/test-files/input-date.html), I've some to the conclusion that we would need to roll out our own date picker. 

Currently, AMP HTML does not have their own date picker, however, there is  currently an [AMP HTML datepicker feature request / issue here](https://github.com/ampproject/amphtml/issues/6469)

I am open to any suggestions.


cc @camelburrito

**Comment by camelburrito** [comment]
This is Working as per browser spec, nothing much we can do here.
